templates/dalite-pre-backup.sh

```diff
@@ -11,4 +11,5 @@ chown "{{ DALITE_LOG_DOWNLOAD_USER }}:{{ MYSQL_DALITE_USER }}" "{{ DALITE_LOG_DO
 chmod g+rwx "{{ DALITE_LOG_DOWNLOAD_LOG_DIR }}" "{{ DALITE_LOG_DOWNLOAD_DB_DIR }}"
 sudo -u "{{ MYSQL_DALITE_USER }}" -H {{DALITE_MANAGE_PY}} dumpdata -o "{{ DALITE_LOG_DOWNLOAD_DB_DIR }}/database-${timestamp}.json"
 mysqldump -u "{{ MYSQL_DALITE_USER }}" -p'{{ MYSQL_DALITE_PASSWORD }}' -h "{{ MYSQL_DALITE_HOST }}" "{{ MYSQL_DALITE_DATABASE }}" -r "{{ DALITE_LOG_DOWNLOAD_DB_DIR }}/database-${timestamp}.sql"
-chown "{{ DALITE_LOG_DOWNLOAD_USER }}:{{ DALITE_LOG_DOWNLOAD_USER }}" "{{ DALITE_LOG_DOWNLOAD_LOG_DIR }}"/* "{{ DALITE_LOG_DOWNLOAD_DB_DIR }}"/*
\ No newline at end of file
+gzip {{ DALITE_LOG_DOWNLOAD_LOG_DIR }}/* {{ DALITE_LOG_DOWNLOAD_LOG_DIR }}/*
```

Comment:
@haikuginger Typo: the second `DALITE_LOG_DOWNLOAD_LOG_DIR` should be `DALITE_LOG_DOWNLOAD_DB_DIR`.

Also, could you please quote the gzip arguments, in case they have spaces or other weird characters in.